app/models/editionable_worldwide_organisation.rb

```diff
@@ -28,6 +28,12 @@ def process_associations_after_save(new_edition)
 
         new_office.contact = office.contact.dup
 
+        office.contact.contact_numbers.each do |contact_number|
+          new_office.contact.contact_numbers << contact_number.dup
+        end
+
+        new_office.save!
```

Comment:
Was this actually saving before, or was the only "saving" happening on `add_office_to_home_page!` below? Struggling to understand how things were persisted before otherwise!

Reply:
I'll update the commit message, but the reason was we weren't saving some records (I think it was only those that weren't being adding to the homepage).